### PR TITLE
Upgrade to node 10

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Upgrade to node 10

### How to review 
Run tests to make sure they pass from this repo and the API

### CODE REVIEW

**Comment by rmccar** [comment]
Not merging until node 10 is in LTS

**Comment by samiwel** [comment]
This should be okay to merge. Node 10 due to become LTS version at the end of October. No need to publish a new version of the schema off the back of this change.

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v10.11.0
```
Comment (tgandrews) [inline .nvmrc]: Could we make this say 10 so that we get all the updates as well and don't have to keep bumping this?

**Comment by rmccar** [comment]
[Review Greenkeeper PR and issues](https://trello.com/c/p2uEzQVy/631-review-greenkeeper-pr-and-issues)

**Comment by samiwel** [comment]
@tgandrews this look good to you now?

**Pull request merged**